webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+
+module.exports = async function (env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv);
```

Comment:
Is this from https://github.com/expo/expo/issues/6660#issuecomment-573506912 ?

If so might be worth adding a comment linking to that so that someone editing this can see the context and understand what can and can't be changed

Reply:
yeah, good idea

Reply:
There's an example of what a generated SVG component looks like here: https://react-svgr.com/docs/getting-started/ (there's also the default template here https://react-svgr.com/docs/custom-templates/ but that's not very readable ).

Since the SVG components are somewhat invisible when generated on the fly this way, it'd be good to include an example of what they look like in the docs.